src/client/Microsoft.Identity.Client/Cache/SuggestedWebCacheKeyFactory.cs

```diff
@@ -16,16 +16,19 @@ public static string GetKeyFromRequest(AuthenticationRequestParameters requestPa
 
             if (requestParameters.ApiId == TelemetryCore.Internal.Events.ApiEvent.ApiIds.AcquireTokenSilent)
             {
-                return requestParameters.Account?.HomeAccountId?.Identifier;
+                return requestParameters.Account?.HomeAccountId?.Identifier == null ? 
```

Comment:
Do we have the risk that the first time the pre-access event is called for OBO, we get a cache key, and the second time we get a different cache key? [we'd want it to be invariant and the same in all calls to before/after access]